### PR TITLE
[EventHubs & AMQP Python] Send Port

### PR DESCRIPTION
**Pull request by yunhaoling**
https://github.com/Azure/azure-sdk-for-python/issues/19235
- port sending
- basic perf/test scripts for sending and receiving

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/azure/eventhub/pyamqp/something.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+b = bytearray(b'\x00S\x12\xd0\x00\x00\x01\n\x00\x00\x00\x0e\xa1$97176fc1-a225-4a22-bdb2-0db19a4aa569CAP\x00P\x00\x00S(\xc0=\x0b\xa10sender-link-97176fc1-a225-4a22-bdb2-0db19a4aa569@@@@@@@@@@\x00S)\xc0B\x07\xa19amqps://pytrack2eh.servicebus.windows.net/eh_for_livetest@@@@@@@@@\x80\x00\x00\x00\x00\x00\x01\x00\x00@@\xc1A\x02\xa3\x19com.microsoft:tracking-id\xa1#a19a961b19c64cf6adba81c5c8c0ffcd_G6')
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/azure/eventhub/pyamqp/something.py:1]: is this file for testing?

**Review by swathipil — COMMENTED** [review]
left a few comments, but in general, LGTM!

File: sdk/eventhub/azure-eventhub/azure/eventhub/pyamqp/something.py (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/azure/eventhub/pyamqp/something.py]: nice catch, it's just a script for local test.
removed

File: sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py [hunk 2]
```diff
@@ -264,7 +270,7 @@ def __init__(self, fully_qualified_namespace, eventhub_name, credential, **kwarg
             self._credential = credential #type: ignore
         self._keep_alive = kwargs.get("keep_alive", 30)
         self._auto_reconnect = kwargs.get("auto_reconnect", True)
-        self._mgmt_target = "amqps://{}/{}".format(
+        self._mgmt_target = "{}/{}".format(
```
[the remark this answers is not in the document]
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py]: I was testing the mgmt link and found that the "amqps://" is prepend by the python amqp impl.

this is a good point regarding to how we construct the uri (protocol "amqps://" in eh or in amqp).
I revert the change first

**Pull request merged**